native/cocos/editor-support/spine-wasm/spine-skeleton-system.cpp

```diff
@@ -10,12 +10,6 @@ void SpineSkeletonSystem::updateAnimation(float deltaTime) {
     auto count = static_cast<int>(vectorSpines.size());
     for (int i = count - 1; i >= 0; --i) {
         SpineSkeletonInstance* spineInstance = vectorSpines[i];
-        if(spineInstance->isDelete) {
-            std::swap(spineInstance, vectorSpines.back());
```

Comment:
spineInstance's value will be set to vectorSpines.back(), and delete spineInstance operation will trigger the last element of the vector to destroy